scripts/lint.sh

```diff
@@ -7,50 +7,28 @@ if [[ $(flutter format -n .) ]]; then
 fi
 
 flutter pub get
-result=$(dartanalyzer lib/)
-if ! echo "$result" | grep -q "No issues found!"; then
-  echo "$result"
-  echo "dartanalyzer issue: lib"
-  exit 1
-fi
+
+# We need to run pubget on all the examples
+for f in doc/examples/**/pubspec.yaml; do
+  d=$(dirname $f)
+  cd $d
+  flutter pub get
+  cd - > /dev/null
```

Comment:
maybe a simpler way is `(cd $d ; flutter pub get)`?

Reply:
I prefer the multiple line on this, bash is not the most easy thing to read, so I think we should every resource that we can to try to make it easier to read. And multiple lines for this seems way easier to read for me.